Kudu.Core/Jobs/JobsManagerBase.cs

```diff
@@ -495,6 +496,16 @@ private static string FindCommandToRun(FileInfoBase[] files, out IScriptHost scr
                         }
                     }
                 }
+
+                foreach (var supportedFileName in scriptHost.SupportedFileNames)
```

Comment:
Cool, thanks. Generally, I look for unit tests by name, or by finding all references to the type, which is why I wasn't finding any. Functional tests are good, but it's ideal to have unit tests as well for everything :) Much easier to run/debug.